search/node-search_test.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
Much easier and clearer to read.   Well done.

Reply:
+1 Nice work here!